kong.cloudformation.yaml

```diff
@@ -900,6 +900,9 @@ Resources:
                         $exec health >> $log
                         if [[ $? -ne 0 ]]; then
                             echo "trying to start kong.."
+
+                            prlimit -n65535 -p $$
```

Comment:
I do see an issue with hardcoding the value here which is not DRY as we also use this value inside of `01_filelimits` command.

Reply:
You can use a Parameter with a default value of 65535 and then just use that in both places with a `!Sub`.